### PR TITLE
scala.js run

### PR DESCRIPTION
**Pull request by rockjam**
Need to add tests

### CODE REVIEW

File: scalajslib/src/mill/scalajslib/ScalaJSModule.scala [hunk 1]
```diff
@@ -89,6 +89,19 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
     )
   }
 
+  override  def runLocal(args: String*) = T.command { run(args:_*) }
+
+  override def run(args: String*) = T.command{
```
Comment (rockjam) [inline scalajslib/src/mill/scalajslib/ScalaJSModule.scala]: Need to pass args

**Comment by lihaoyi** [comment]
Looks good to me

File: scalajslib/src/mill/scalajslib/ScalaJSModule.scala (hunk 1 above)
Comment (rockjam) [reply scalajslib/src/mill/scalajslib/ScalaJSModule.scala]: Actually scala.js doesn't pass args in sbt

**Comment by rockjam** [comment]
@lihaoyi added some tests and `runMain` command too

**Pull request merged**